src/data/equiv/basic.lean

```diff
@@ -8,7 +8,7 @@ We say two types are equivalent if they are isomorphic.
 
 Two equivalent types have the same cardinality.
 -/
-import logic.function logic.unique data.set.basic data.bool data.quot
+import tactic.split_ifs logic.function logic.unique data.set.basic data.bool data.quot
```

Comment:
Can `tactic.split_ifs` be imported into `tactic.interactive` so this isn't necessary? `tactic.split_ifs` has no prerequisites and seems like a good global tactic

Reply:
I think it already is imported by `tactic.interactive`.